code/game/machinery/computer/arcade.dm

```diff
@@ -371,8 +371,10 @@ GLOBAL_LIST_INIT(arcade_prize_pool, list(
 		if(0 to 2)
 			LAZYADD(last_three_move, player_stance)
 		if(3)
-			LAZYREMOVE(last_three_move, last_three_move[1])
-			LAZYADD(last_three_move, player_stance)
+			for(var/i = 1; i < LAZYLEN(last_three_move); i++)
+				last_three_move[i] = LAZYACCESS(last_three_move, i + 1)
+			last_three_move[LAZYLEN(last_three_move)] = player_stance
```

Comment:
You don't need to use the lazylist macros, since we know both that the list exists and its exact length thanks to the switch.
```
for(var/i in 1 to 2)
	last_three_move[i] = last_three_move[i + 1]
last_three_move[3] = player_stance
```